data/json/items/magazines.json

```diff
@@ -71,5 +71,218 @@
+        "id" : "m1918mag",
+        "//" : "Compatable magazines made for reproductions exist, and the standard-capacity mags are fairly cheap compared to variants.",
+        "price" : 3600,
+        "name" : "BAR 20-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with the Browning Automatic Rifle, holds 20 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 198,
+        "to_hit" : -2,
+        "ammo_type" : "3006",
+        "capacity" : 20,
+        "reliability" : 9,
+        "reload_time" : 30
+    },{
+        "type" : "MAGAZINE",
+        "id" : "m1918mag_30",
+        "price" : 8500,
+        "name" : "BAR 30-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An aftermarket magazine for use with the Browning Automatic Rifle, holds 30 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 297,
+        "to_hit" : -2,
+        "ammo_type" : "3006",
+        "capacity" : 30,
+        "reliability" : 8,
+        "reload_time" : 35
+    },{
+        "type" : "MAGAZINE",
+        "id" : "ak74_mag_30",
+        "price" : 1500,
+        "name" : "AK-74M 30-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard magazine made for the AK-74M rifle, made of steel-reinforced polymer.",
+        "material" : ["steel", "plastic"],
+        "volume" : 2,
+        "weight" : 230,
+        "to_hit" : -2,
+        "ammo_type" : "545x39",
+        "capacity" : 30,
+        "reliability" : 9,
+        "reload_time" : 30
+    },{
+        "type" : "MAGAZINE",
+        "id" : "rpk_74_mag_45",
+        "price" : 8500,
+        "//" : "Technically it's meant for the RPK-74, but can always rename if we add an RPK.",
+        "name" : "AK-74M 45-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A 45-round magazine compatible with the AK-74M rifle, though originally designed for the RPK-74 light machine gun.",
+        "material" : "plastic",
+        "volume" : 3,
+        "weight" : 300,
+        "to_hit" : -2,
+        "ammo_type" : "545x39",
+        "capacity" : 45,
+        "reliability" : 8,
+        "reload_time" : 40
+    },{
+        "type" : "MAGAZINE",
+        "id" : "belt308",
+        "price" : 0,
+        "name" : "M13 7.62x51mm ammo belt",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "An ammo belt consisting of metal linkages which disintegrate upon firing.  Designed for the M60 machine gun, and additionally compatible with the M240 machine gun.",
+        "material" : "steel",
+        "volume" : 0,
+        "weight" : 0,
+        "to_hit" : -4,
+        "rigid": false,
+        "ammo_type" : "308",
+        "capacity" : 200,
+        "count" : 100,
+        "reliability" : 8,
+        "reload_time" : 80,
```

Comment:
Yes, updated since then. I'll post an update to #14981 if I make further changes to the JSON specifications.